tests/provider: Remove ID-Only Refresh Testing (Fix and Enable tfproviderlint AT010 and AT011)

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Description

Identifier only refresh testing is generally a legacy testing practice before full import testing was the normal convention and the testing framework automatically performed refresh after apply. Other use cases for `IDRefreshName` are not clear. Given this configuration can cause unexpected testing issues when combined with other advanced testing scenarios and it is not commonly used, remove the remaining usage of `helper/resource.TestCase` type `IDRefreshName` and `IDRefreshIgnore` fields. `tfproviderlint` checks will be added to help enforce this going forward.

### Affected Files

```
aws/resource_aws_alb_target_group_test.go
aws/resource_aws_appautoscaling_target_test.go
aws/resource_aws_codecommit_repository_test.go
aws/resource_aws_customer_gateway_test.go
aws/resource_aws_default_route_table_test.go
aws/resource_aws_default_security_group_test.go
aws/resource_aws_ebs_volume_test.go
aws/resource_aws_ec2_carrier_gateway_test.go
aws/resource_aws_egress_only_internet_gateway_test.go
aws/resource_aws_eip_test.go
aws/resource_aws_elastic_transcoder_pipeline_test.go
aws/resource_aws_elb_attachment_test.go
aws/resource_aws_elb_test.go
aws/resource_aws_iam_group_policy_test.go
aws/resource_aws_iam_instance_profile_test.go
aws/resource_aws_iam_role_policy_test.go
aws/resource_aws_iam_role_test.go
aws/resource_aws_iam_user_policy_test.go
aws/resource_aws_instance_test.go
aws/resource_aws_internet_gateway_test.go
aws/resource_aws_key_pair_test.go
aws/resource_aws_lb_listener_test.go
aws/resource_aws_lb_target_group_test.go
aws/resource_aws_lb_test.go
aws/resource_aws_lightsail_instance_test.go
aws/resource_aws_nat_gateway_test.go
aws/resource_aws_network_acl_test.go
aws/resource_aws_network_interface_attachment_test.go
aws/resource_aws_network_interface_test.go
aws/resource_aws_pinpoint_adm_channel_test.go
aws/resource_aws_pinpoint_apns_channel_test.go
aws/resource_aws_pinpoint_apns_sandbox_channel_test.go
aws/resource_aws_pinpoint_apns_voip_channel_test.go
aws/resource_aws_pinpoint_apns_voip_sandbox_channel_test.go
aws/resource_aws_pinpoint_app_test.go
aws/resource_aws_pinpoint_baidu_channel_test.go
aws/resource_aws_pinpoint_email_channel_test.go
aws/resource_aws_pinpoint_event_stream_test.go
aws/resource_aws_pinpoint_gcm_channel_test.go
aws/resource_aws_pinpoint_sms_channel_test.go
aws/resource_aws_route53_delegation_set_test.go
aws/resource_aws_route53_health_check_test.go
aws/resource_aws_route53_record_test.go
aws/resource_aws_route_table_test.go
aws/resource_aws_security_group_test.go
aws/resource_aws_sns_topic_test.go
aws/resource_aws_subnet_test.go
aws/resource_aws_transfer_server_test.go
aws/resource_aws_transfer_user_test.go
aws/resource_aws_vpc_endpoint_connection_notification_test.go
aws/resource_aws_vpc_endpoint_test.go
aws/resource_aws_vpn_connection_test.go
aws/resource_aws_vpn_gateway_route_propagation_test.go
aws/resource_aws_vpn_gateway_test.go
```

### References

* https://github.com/hashicorp/terraform-provider-aws/pull/18349
* https://github.com/hashicorp/terraform-plugin-sdk/issues/730
* https://github.com/bflad/tfproviderlint/issues/228

This is now covered by `tfproviderlint` `AT010` and `AT011` checks, which can be fixed and enabled at any point.

```
aws/resource_aws_alb_target_group_test.go:54:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:97:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:118:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:140:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:169:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:198:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:227:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:254:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:286:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:349:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:435:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_alb_target_group_test.go:464:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_appautoscaling_target_test.go:24:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_appautoscaling_target_test.go:87:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_codecommit_repository_test.go:145:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_customer_gateway_test.go:24:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_customer_gateway_test.go:60:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_customer_gateway_test.go:103:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_customer_gateway_test.go:134:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_customer_gateway_test.go:188:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_default_route_table_test.go:94:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_default_route_table_test.go:162:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_default_route_table_test.go:301:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_default_security_group_test.go:21:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_default_security_group_test.go:75:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:84:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:120:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:156:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:192:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:228:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:264:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:303:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:391:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:420:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:449:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:477:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:516:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:573:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:630:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:688:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ebs_volume_test.go:728:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ec2_carrier_gateway_test.go:78:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ec2_carrier_gateway_test.go:109:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_ec2_carrier_gateway_test.go:133:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_egress_only_internet_gateway_test.go:97:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:123:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:152:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:179:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:210:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:247:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:338:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:432:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:461:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:491:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:526:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_eip_test.go:601:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elastic_transcoder_pipeline_test.go:25:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elastic_transcoder_pipeline_test.go:54:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elastic_transcoder_pipeline_test.go:83:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elastic_transcoder_pipeline_test.go:154:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elastic_transcoder_pipeline_test.go:188:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_attachment_test.go:20:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_attachment_test.go:81:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:74:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:135:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:158:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:199:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:239:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:262:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:285:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:307:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:337:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:426:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:465:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:495:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:623:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:649:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:677:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:698:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:724:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:747:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_elb_test.go:781:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_iam_group_policy_test.go:91:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_iam_group_policy_test.go:132:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_iam_instance_profile_test.go:78:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_iam_role_policy_test.go:105:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_iam_role_policy_test.go:151:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_iam_role_test.go:225:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_iam_user_policy_test.go:99:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_iam_user_policy_test.go:142:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:244:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:274:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:404:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:449:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:541:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:618:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:673:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:730:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:790:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:825:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:856:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:885:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:985:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1013:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1051:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1283:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1340:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1405:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1443:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1484:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1527:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:1588:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:3352:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_instance_test.go:3950:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_internet_gateway_test.go:146:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_internet_gateway_test.go:193:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_internet_gateway_test.go:218:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_key_pair_test.go:173:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:24:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:58:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:130:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:162:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:195:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:277:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:314:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:350:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_listener_test.go:388:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:102:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:148:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:180:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:226:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:252:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:366:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:421:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:477:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:508:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:576:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:619:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:641:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:664:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:694:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:724:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:754:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:782:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:823:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:861:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:934:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:1021:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:1083:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:1132:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:1174:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:1198:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_target_group_test.go:1258:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:103:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:141:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:285:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:355:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:385:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:420:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:442:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:464:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:489:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:530:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:572:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:613:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:655:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:697:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:728:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:759:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:792:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:826:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:915:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:986:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:1075:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lb_test.go:1145:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lightsail_instance_test.go:86:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lightsail_instance_test.go:122:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_lightsail_instance_test.go:169:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_nat_gateway_test.go:66:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:313:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:356:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:391:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:447:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:473:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:502:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:548:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:597:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:634:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:691:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_acl_test.go:722:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_attachment_test.go:19:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:77:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:112:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:160:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:206:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:276:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:310:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:339:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:368:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_network_interface_test.go:409:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_adm_channel_test.go:52:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_apns_channel_test.go:99:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_apns_channel_test.go:134:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_apns_sandbox_channel_test.go:99:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_apns_sandbox_channel_test.go:134:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_apns_voip_channel_test.go:99:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_apns_voip_channel_test.go:134:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_apns_voip_sandbox_channel_test.go:99:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_apns_voip_sandbox_channel_test.go:134:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_app_test.go:74:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_app_test.go:101:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_app_test.go:130:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_app_test.go:159:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_baidu_channel_test.go:24:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_email_channel_test.go:20:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_event_stream_test.go:23:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_gcm_channel_test.go:33:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_sms_channel_test.go:20:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_sms_channel_test.go:66:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_pinpoint_sms_channel_test.go:117:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_delegation_set_test.go:23:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_delegation_set_test.go:56:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_health_check_test.go:21:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_health_check_test.go:58:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_health_check_test.go:102:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_health_check_test.go:265:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:118:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:222:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:264:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:291:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:319:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:347:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:375:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:401:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:428:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:463:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:491:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:554:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:641:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:668:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:735:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:768:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:829:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:864:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:900:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:935:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:970:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route53_record_test.go:992:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:234:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:285:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:321:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:360:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:405:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:424:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:744:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:780:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_route_table_test.go:924:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:689:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1062:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1243:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1279:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1324:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1362:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1399:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1426:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1460:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_security_group_test.go:1573:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:96:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:123:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:150:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:180:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:208:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:227:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:248:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:278:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:312:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_sns_topic_test.go:347:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_subnet_test.go:133:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_subnet_test.go:171:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_subnet_test.go:575:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_subnet_test.go:616:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_subnet_test.go:660:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_subnet_test.go:804:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_server_test.go:75:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_server_test.go:122:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_server_test.go:166:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_server_test.go:220:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_server_test.go:254:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_server_test.go:284:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_user_test.go:23:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_user_test.go:56:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_transfer_user_test.go:177:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpc_endpoint_connection_notification_test.go:22:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpc_endpoint_test.go:484:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_connection_test.go:103:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_connection_test.go:213:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_connection_test.go:322:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_connection_test.go:351:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_connection_test.go:379:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_connection_test.go:406:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_gateway_route_propagation_test.go:19:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_gateway_test.go:113:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_gateway_test.go:261:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_gateway_test.go:326:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
aws/resource_aws_vpn_gateway_test.go:355:3: AT010: prefer TestStep ImportState testing over TestCase IDRefreshName
```

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -94,7 +94,6 @@ awsproviderlint:
 		-AWSAT006=false \
 		-AWSR002=false \
 		-AWSV001=false \
-		-AT010=false \
 		-R001=false \
 		-R010=false \
 		-R018=false \

--- a/aws/resource_aws_eip_test.go
+++ b/aws/resource_aws_eip_test.go
@@ -666,11 +666,10 @@ func TestAccAWSEIP_BYOIPAddress_default(t *testing.T) {
 	resourceName := "aws_eip.test"
 
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:      func() { testAccPreCheck(t) },
-		ErrorCheck:    testAccErrorCheck(t, ec2.EndpointsID),
-		IDRefreshName: resourceName,
-		Providers:     testAccProviders,
-		CheckDestroy:  testAccCheckAWSEIPDestroy,
+		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheck(t, ec2.EndpointsID),
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSEIPDestroy,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccAWSEIPConfig_BYOIPAddress_custom_default,
@@ -696,11 +695,10 @@ func TestAccAWSEIP_BYOIPAddress_custom(t *testing.T) {
 	address := os.Getenv("AWS_EC2_EIP_BYOIP_ADDRESS")
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:      func() { testAccPreCheck(t) },
-		ErrorCheck:    testAccErrorCheck(t, ec2.EndpointsID),
-		IDRefreshName: resourceName,
-		Providers:     testAccProviders,
-		CheckDestroy:  testAccCheckAWSEIPDestroy,
+		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheck(t, ec2.EndpointsID),
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSEIPDestroy,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccAWSEIPConfig_BYOIPAddress_custom(address),
@@ -731,11 +729,10 @@ func TestAccAWSEIP_BYOIPAddress_custom_with_PublicIpv4Pool(t *testing.T) {
 	poolName := os.Getenv("AWS_EC2_EIP_PUBLIC_IPV4_POOL")
 
 	resource.Test(t, resource.TestCase{
-		PreCheck:      func() { testAccPreCheck(t) },
-		ErrorCheck:    testAccErrorCheck(t, ec2.EndpointsID),
-		IDRefreshName: resourceName,
-		Providers:     testAccProviders,
-		CheckDestroy:  testAccCheckAWSEIPDestroy,
+		PreCheck:     func() { testAccPreCheck(t) },
+		ErrorCheck:   testAccErrorCheck(t, ec2.EndpointsID),
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSEIPDestroy,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccAWSEIPConfig_BYOIPAddress_custom_with_PublicIpv4Pool(address, poolName),

